doc/migrate-to-pymongo4.rst

```diff
@@ -725,3 +725,9 @@ pymongo.message helpers are removed
 Removed :meth:`pymongo.message.delete`, :meth:`pymongo.message.get_more`,
 :meth:`pymongo.message.insert`, :meth:`pymongo.message.kill_cursors`,
 :meth:`pymongo.message.query`, and :meth:`pymongo.message.update`.
+
+
+Name is a required argument for pymongo.driver_info.DriverInfo
+..............................................................
+``name`` is now a required argument for the :class:`pymongo.driver_info.DriverInfo` class.
```

Comment:
I think there needs to be a blank line after the `.......` header.

Reply:
Done.